eo2js/src/commands/transpile.js

```diff
@@ -47,13 +47,11 @@ const makeDirIfNotExist = function(dir) {
  */
 const hasMeta = function(xmir, name) {
   const metas = xmir.program.metas.meta
-  let res = false
   if (Array.isArray(metas)) {
-    res = metas.findIndex((meta) => meta.head === name) !== -1
-  } else if (typeof metas === 'object' && metas.hasOwnProperty('head')) {
-    res = metas.head === name
+    return metas.some((meta) => meta.head === name);
   }
-  return res
+  return metas?.head === name;
```

Comment:
@Akvadevka we're trying to avoid many `return` statements, [here's](https://www.yegor256.com/2015/08/18/multiple-return-statements-in-oop.html) why